### DISCUSSION TITLE
Uvs are flipped when sent to ZB

### DISCUSSION THREAD

**Discussion by canderled** [JoseConseco/GoB#384] [Q&A]
Hello guys, is there an option or a way to avoid aving the UVs being flipped when sent to and imported from ZB using the GOB ? this is specially annoying when working with udim. 

![UVFlipped](https://user-images.githubusercontent.com/96201702/206429696-c2314239-d7f9-418c-b008-0c47ae22e80a.jpg)

**Comment by goodstout** [comment]
late, late reply. but this is actually kind of a weird Zbrush issue. Zbrush flips all its UV's, so they're likely having to do some conversion along the way?

**Comment by gryzka** [comment]
![fliped UDIMs](https://github.com/user-attachments/assets/d3d58c54-c7d4-41bd-98ab-ec92ddc523be)
Yeah, I agree with Canderled. Please don't flip UVs via GoB. This practice makes working with displacement maps in ZBrush much more complicated. When you have a character with many flipped UDIMs, baking displacement maps in ZBrush causes incorrect UDIM naming, complicating our work in the VFX and game pipeline. I noticed that only Maya GoZ works without flipping UVs, which is crucial. Other GoZ versions, like MODO and Blender, are flipping the UVs. Could you add an option dont flip UV?

**Comment by gryzka** [comment]
Is there any chance to have this option in an upcoming update? This is the most important feature for us here in the VFX and game development industry. Is it possible to support you financially to speed up the development of this feature? Thank you!

**Reply by kromar** [reply]
> Is there any chance to have this option in an upcoming update? This is the most important feature for us here in the VFX and game development industry. Is it possible to support you financially to speed up the development of this feature? Thank you!

Hi, @gryzka i have missed this discussion. I created an issue about this topic which you can follow here: https://github.com/JoseConseco/GoB/issues/494

Of course any financial support is more than welcome. For one time or recurring donations you can find a link in my github profile or go directly to this link: https://ko-fi.com/kromar
But if this is urgent for you, I'm happy to discuss what you have on your mind of course. Feel free to reach out to me via mail (3d.kromar at gmail dot com) or on my discord channel which you can find in the github profile as well.

**Comment by kromar** [comment]
uv flipping has been added to the latest release https://github.com/JoseConseco/GoB/releases/tag/v4_1_2

**Reply by gryzka** [reply]
Awesome news! I’m not sure where to find this option. I tested it without changing any settings, but my UVs are still flipped.

**Reply by kromar** [reply]
in the addon preferences you should find options for the flipping
![image](https://github.com/user-attachments/assets/66aa7375-a51d-4894-a920-9c5b07d7645e)

**Reply by gryzka** [reply]
I didn’t see this option at first because I didn’t restart Blender after removing the previous version. Now the option is available, and it works perfectly! This is probably the most important update for me! Many thanks!
